google/cloud/storage/quickstart/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@
 # configuration files are usable.
 
 cmake_minimum_required(VERSION 3.5)
-project(google-cloud-cpp-storage-install-integration-test CXX C)
+project(google-cloud-cpp-storage-quickstart CXX C)
```

Comment:
This and the bigtable/quickstart/CMakeLIsts.txt file also contain documentation that says "A minimal CMakeLists.txt to *test* the install ...". May want to update that doc comment too to make it clearer that this file is to serve as an example for our users, not for us to test our installs.

Reply:
Fixed here and the WORKSPACE files too.